lab-anthony/lib/handle-error.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
This does nothing with the messages you have painstakingly added to each of your errors.